spec/views/notifications/index.html.erb_spec.rb

```diff
@@ -1,6 +1,7 @@
 require "rails_helper"
 
 RSpec.describe "notifications/index", type: :view do
+  let(:base_time) { Date.new(2025, 1, 2) }
```

Comment:
what's going on here? seems unrelated?

Reply:
yeah this is unrelated. fixes a flaky test (from a previous PR of mine)